aoe4/formatters.js

```diff
@@ -124,13 +129,20 @@ class NightbotDefaultFormatter {
 
     const rank = mode.rank ? `rank #${mode.rank}` : 'unranked';
     const rank_level = mode.rank_level ? formatRankLevel(mode.rank_level) + ', ' : '';
-    var msg = `${player.name} is ${rank} (${rank_level}${mode.rating} Elo), with ${mode.games_count} game${mode.games_count == 1?'':'s'} (${mode.wins_count}-${mode.losses_count} | ${mode.win_rate}%)`;
+    const rating = player.rating ? ` (${rank_level}${mode.rating} Elo)` : '';
+    const games = mode.games_count ? mode.games_count : "no";
+    var msg = `${player.name} is ${rank}${rating}, with ${games} game${mode.games_count == 1?'':'s'} (${mode.wins_count?mode.wins_count:0}-${mode.losses_count?mode.losses_count:0} | ${mode.win_rate?mode.win_rate:0}%)`;
```

Comment:
Put win rate in a separate if statement with ` msg +=` so that ` (win-loss | 0%)`  is completely omitted when games_count == 0. Coz there's no point in `(0-0 | 0%)`. Just leave it out entirely.

Reply:
This is how it appeared in the issue, but will do